frontend/admin/src/js/components/Table/Table.tsx

```diff
@@ -37,6 +37,7 @@ export interface TableProps<
   pagination?: boolean;
   hiddenCols?: string[];
   labelledBy?: string;
+  sortBy?: Array<{ id: string; desc: boolean }>;
```

Comment:
Could we rename this to something like `initialSortBy` to be more explicit about that it does?

Reply:
See: [Update Table prop name initialSortBy](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5116/commits/73658f6bdfafaa8b7c56a334a2c11a05b2341dcb).